resfulservice/src/routes/curation.js

```diff
@@ -19,4 +19,7 @@ router.route('/admin')
 router.route('rehydrate')
   .patch(isAuth, curationController.curationRehydration);
 
+router.route('/getdoi/:doi([^/]*)')
+  .get(curationController.getDoiInfo);
+
```

Comment:
I would recommend relocating this to the knowledge route since it involves fetching data from an external service via an API call, and aligning it with the other knowledge-related functionalities.

Reply:
Definitely, I can move it. We originally discussed putting it in curation on the call so that's why I put it here originally